src/main/java/com/gistpetition/api/petition/application/PetitionService.java

```diff
@@ -95,7 +94,9 @@ public void deletePetition(Long petitionId) {
     public void agree(AgreementRequest request, Long petitionId, Long userId) {
         Petition petition = findPetitionById(petitionId);
         User user = findUserById(userId);
-        petition.applyAgreement(user, request.getDescription());
+        Agreement agreement = new Agreement(request.getDescription(), user.getId());
```

Comment:
```suggestion
        Agreement agreement = new Agreement(request.getDescription(), user.getId(), petition);
```
이렇게 agreement 생성시 petition을 넣지 않고, setter로 넣는 이유가 있을까요?

라고 생각하고 아래를 봤더니 편의 메서드를 정의하셨군요!